File: 백준/Silver/1764. 듣보잡/듣보잡.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n');
const [N, M] = input[0].split(' ').map(Number);
const stranger = new Set();
const answer = [];

for (let i = 1; i <= N; i++) {
    stranger.add(input[i]);
}

for (let j = N + 1; j < input.length; j++) {
    if (stranger.has(input[j])) {
        answer.push(input[j]);
    }
}

console.log(`${answer.length}\n${answer.sort().join('\n')}`);

File: 백준/Silver/1764. 듣보잡/듣보잡.py
import sys

input = sys.stdin.readlines()
N, M = map(int, input[0].split())
A = set(map(lambda x: x.strip(), input[1:N+1]))
B = set(map(lambda x: x.strip(), input[N+1:]))
C = A & B
print(len(C))
print('\n'.join(sorted(map(str, C))))